src/main/java/core/basesyntax/strategy/operation/ReturnOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+
+public class ReturnOperation implements OperationHandler {
+    private StorageDao storageDao = new StorageDaoImpl();
+
+    @Override
+    public void operationType(String fruit, int amount) {
```

Comment:
The method name `operationType` is not descriptive of the action performed within the method. Consider renaming it to something more indicative of the operation, like `applyReturnOperation` or `handleReturn`.